plugins/tab-one/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "tab-one",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "author": "",
+    "open-edx-meta": {
+        "tabs": {
+            "name": "Tab One",
+            "help": "This is Tab One",
+            "order": 1
+        }
+    },
```

Comment:
We can create some standardised metadata format that will allow specifying extra data for each plugin. 

I don't think it should be used the way I've used it though, because that will make translation difficult. Instead the plugin component can directly export these as via a known name. i.e. 

```javascript
export const tabName = intl.formatMessage("Tab One") // illustrative code
```